lib/routes/questions.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable radix */
+/* eslint-disable linebreak-style */

```

Comment:
Please use es6 imports

Reply:
es6 imports has been included. i just had to install babel